opm/simulators/wells/StandardWellConnections.cpp

```diff
@@ -459,29 +460,24 @@ computeDensitiesForStoppedProducer(const DensityPropertyFunctions& prop_func)
     }
 }
 
-template<class FluidSystem, class Indices>
+template<typename FluidSystem, typename Indices>
 typename StandardWellConnections<FluidSystem, Indices>::Properties
-StandardWellConnections<FluidSystem,Indices>::
-computePropertiesForPressures(const WellState<Scalar>&         well_state,
+StandardWellConnections<FluidSystem, Indices>::
+computePropertiesForPressures(const WellState<Scalar, IndexTraits>&         well_state,
                               const PressurePropertyFunctions& prop_func) const
 {
     auto props = Properties{};
 
     const int nperf = well_.numLocalPerfs();
-    const PhaseUsage& pu = well_.phaseUsage();
 
     props.b_perf        .resize(nperf * this->well_.numComponents());
     props.surf_dens_perf.resize(nperf * this->well_.numComponents());
 
     const auto& ws = well_state.well(this->well_.indexOfWell());
 
-    static constexpr int Water = BlackoilPhases::Aqua;
-    static constexpr int Oil = BlackoilPhases::Liquid;
-    static constexpr int Gas = BlackoilPhases::Vapour;
-
-    const bool waterPresent = FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx);
-    const bool oilPresent = FluidSystem::phaseIsActive(FluidSystem::oilPhaseIdx);
-    const bool gasPresent = FluidSystem::phaseIsActive(FluidSystem::gasPhaseIdx);
+    const bool waterPresent = FluidSystem::phaseIsActive(IndexTraits::waterPhaseIdx);
```

Comment:
there are a mixed usage of `IndexTraits::***PhaseIdx`. was thinking all changing to `FluidSystem::***PhaseIdx`, while now thinking maybe all should be `IndexTraits::**PhaseIdx`, the other way around.